core/services/chainlink/relayer_factory.go

```diff
@@ -268,12 +269,12 @@ func (c CosmosFactoryConfig) Validate() error {
 	return err
 }
 
-func (r *RelayerFactory) NewCosmos(config CosmosFactoryConfig) (map[types.RelayID]CosmosLoopRelayerChainer, error) {
+func (r *RelayerFactory) NewCosmos(config CosmosFactoryConfig) (map[types.RelayID]loop.Relayer, error) {
```

Comment:
Could this just use `NewLOOPRelayer`?

Reply:
Hmmm what do you mean? This is a 'universal' relayer factory, not just a Cosmos relayer factory.

Reply:
Oh I see, is that a helper? Looks like we could switch to it after we remove the in-memory version?